### PR TITLE
[Android] Display suggestion banner in the app

### PR DESCRIPTION
**Pull request by darcywong00**
This PR adjusts the Keyman keyboard to display the suggestion banner (`KMManager.java`).

atm, there's test code to have KMW set the banner to `'suggestion'` and use some canned suggestion words.

Screenshot of Pixel 2 emulator
Screenshot_1554224835

Screenshot of Nexus 10 emulator
Screenshot_1554225057

Changes
* Allow embedded app to pass a banner height to the banner constructor
* Escape single-quotes when registering lexical models (comment from a previous PR)

TODO:
- [ ] Investigate the factor of `-48` in adjusting the banner height in banner.ts
- [x] Adjust the banner width when rotating keyboard
- [ ] Adjust the Keyboard height when the banner toggles between blank / suggestion

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/149024:id/kvk/index.html)

File: android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java [hunk 1]
```diff
@@ -668,8 +670,9 @@ public static boolean registerLexicalModel(HashMap<String, String> lexicalModelI
       return false;
     }
 
-    // Use single quotes in JSONobject for javascript call
+    // Escape single quotes, and then convert double quotes to single quotes for javascript call
     String model = String.valueOf(modelObj);
+    model = model.replaceAll("\'", "\\\\'"); // Double-escaped-backslash b/c regex.
```
Comment (mcdurdin) [inline android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java]: Is that supposed to replace `\'` with `\\'`? If so, you need to have:
```suggestion
    model = model.replaceAll("\\'", "\\\\'"); // Double-escaped-backslash b/c regex.
```

Comment (darcywong00) [reply android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java]: The code is lifted from KMKeyboard.java ll.350-351 for escaping single-quoted keyboard and language names
```java
    // Escape single-quoted names for javascript call
    keyboardName = keyboardName.replaceAll("\'", "\\\\'"); // Double-escaped-backslash b/c regex.
    languageName = languageName.replaceAll("\'", "\\\\'");
```
It looks to replace a literal `'` with a double-escaped backslash

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/149042:id/kvk/index.html)

**Comment by darcywong00** [comment]
I guess I'll go ahead merge this and resolve the the TODO's in a future PR.

And @jahorton , I promise I will remove the test code in KMW

**Pull request merged**